### PR TITLE
[READY] THE LONG AWAITED BLINDNESS QUIRK PR

### PR DESCRIPTION
**Pull request by Militaires**
image

blindness is now a quirk you can choose, it gives you 4 points, the highest of any quirk (before 3 points), it also gives you a unique blindfold that lets people easily identify you as a blind person, this blindfold matches your eye color. surgery can not restore your vision either

:cl: Basilman
add: Added the blindness quirk, good luck.
/:cl:

[why]: # because it's an interesting quirk for people who wanna play the game in a much more different way, also EMERGENT gameplay when you steal blind people's blindfolds to disguise yourself as a blind person

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
I will unironically use this quirk

**Comment by Militaires** [comment]
@anconfuzedrock yeah me too, i knew you'd love it because of lilith

**Comment by Militaires** [comment]
im gonna make it so that u cant take nearsighted wit this real quick

**Comment by Tlaltecuhtli** [comment]
will antags like wiz, nuke ops keep it?

**Comment by Lxtoofonea** [comment]
+1

**Comment by wesoda25** [comment]
Long have I waited

**Comment by PKPenguin321** [comment]
quad amputee quirk when?

**Comment by optimumtact** [comment]
@Lxtoofonea please use emojii instead of leaving a one word comment

**Comment by Lxtoofonea** [comment]
@optimumtact sorry bout that, was using my phone to do it.

i do think this is a very good idea, we also need to add more good quirks as well, right now you can get more points then you can spend

**Comment by anconfuzedrock** [comment]
they just won't work hathkar if you've ever been blind in game you know how it works

**Comment by MrDoomBringer** [comment]
holy fuck im doing this shit

**Comment by Anonymous583** [comment]
blind plasmaman will be the penulimate hard mode

**Comment by anconfuzedrock** [comment]
why not just have them start with no eyes at all

**Comment by GunHog** [comment]
Will replacing the eyes via surgery work to restore sight?

**Comment by kingoftheautism** [comment]
probably going to play blind miner, good pr

**Comment by optimumtact** [comment]
@GunHog please read the PR OP it does discuss surgery in the final sentence

**Comment by Bawhoppen** [comment]
I disagree with it giving four points. Maybe make it give only one or two like usual. This isn't something you're required to have, so there's no obligation to balance it like you are.

**Comment by MrDoomBringer** [comment]
@Bawhoppen literally what? none of the quirks are something you're required to have

thats why theyre quirks

**Comment by Bawhoppen** [comment]
@MrDoomBringer That's my point. You don't choose negative ones because they're efficient at giving you points for your positive quirks. You choose them because they're a fun gimmick/RP element/etc.

**Comment by Keluandrie** [comment]
@Bawhoppen I don't think you understand how much of a game over permanent blindness really is. Nobody's going to pick this shit for more points, they'll pick it only for the challenge.

**Comment by Bawhoppen** [comment]
To clarify, the reason I care it is worth four points, is the precedent it sets for the future. I personally think the max point gain from negatives should be two, even if it's much more negative than some of the other +2 traits.

**Comment by anconfuzedrock** [comment]
lmao baw there's a brain tumour that gives you three people can get a free backpack size toolbox and get a point who gives a shit about the +4 points

**Comment by Bawhoppen** [comment]
@anconfuzedrock Yeah, I understand. It's just the last thing I want to see is the trait system being powergamed.

**Comment by IllotAFV** [comment]
I dont think,that super Hi-Tech research facility will hire blind people to work on it.

**Comment by Anonmare** [comment]
Diversity hiring is getting out of hand

**Comment by XDTM** [comment]
This is the same corporation that hires clowns and mimes though

**Comment by subject217** [comment]
it also "hires" people who intentionally antagonize the people who actually have jobs so the deep, deep lore will survive a blindness quirk

**Comment by Buggy123** [comment]
If people don't want it to be +4, maybe reduce it to +2, but also have them spawn with glasses that cure their blindness if they're wearing it, and have no other benefits or makes them more vulnerable to flashes?
image

**Comment by subject217** [comment]
We already have a nearsighted quirk, Mr. Hasn't played in 7 months.

**Comment by anconfuzedrock** [comment]
Kinda defeats the point of the quirk buggy.
I don't think we gotta worry about people powergaming blindness with what it is, but I personally can't see how it'll set a precedent. if blindness gives four points, it'll just tell people "unless your quirk gives you liver cancer don't give it four points" 
If I were worried about powergaming quirks, I'd be more worried about the cheap mostly harmless ones. like every assistant I ever search has a shitty fucking toolbox.

**Comment by XDTM** [comment]
I know it's a balance point and all, but it's weird how quirks are absolutely unhealable when both brain-based and damage-based blindness can exist and be cured in game.

**Comment by Militaires** [comment]
@Iamgoofball @GunHog read the pr

chems shouldnt work on this either because it uses the same system that the nearsighted quirk uses to disable healing

**Comment by ExcessiveUseOfCobblestone** [comment]
> powergaming being able to see 1 tile away as opposed to 7

lol'd

**Comment by MrDoomBringer** [comment]
This quirk should spawn you with some kinda unique bandanna or something, so people know you're the real blind deal.

**Comment by ExcessiveUseOfCobblestone** [comment]
did you read OP

**Comment by ExcessiveUseOfCobblestone** [comment]
> I know it's a balance point and all, but it's weird how quirks are absolutely unhealable when both brain-based and eye-based blindness can exist and be cured in game.

I agree, it is pretty immersion breaking you can have your eye stabbed to mush and back but someone who starts blind just magically can't be un-blinded.

**Comment by MrDoomBringer** [comment]
@ExcessiveUseOfCobblestone blindfolds already exist in game u square, i mean a special one for the quirk. Like how assistants have a special toolbox for their heirloom

**Comment by ExcessiveUseOfCobblestone** [comment]
> a **unique** blindfold that lets people easily **identify you as a blind person**, **this blindfold matches your eye color**

You mean like a unique toolbox that lets people easily identify you as a heirloom person?

**Comment by Militaires** [comment]
normal blindfolds are all black, these are colored depending on your guy's eyes

**Comment by Militaires** [comment]
also everyone here citing emulsions can eat my entire gay

**Comment by AlcaroIsAFrick** [comment]
u could have used blur filters

**Comment by obscolene** [comment]
>4 points is too much
Mother Fucker if this shit gave 20 points it would still be complete shit.

**Comment by optimumtact** [comment]
It makes sense to be unhealable.

If you wdon't want to be blind, don't take the quirk, we can't make it healable because then people will just take it for the quirk points and heal themselves at roundstart.

**Comment by MrDoomBringer** [comment]
Making the negative quirks healable is retarded - might as well put the positive quirks on a timer or some shit then

**Comment by XDTM** [comment]
Yeah i did say that i understood the balance reasons for it, i just wish there was a way to make them consistent with the rest of the game without them being free points.

Maybe we could scrap the point system, give everyone one free positive quirk, X neutral quirks, and unlimited negative quirks?

**Comment by Militaires** [comment]
@mrdoombringer you cant color blindfolds with crayons what are you on

**Comment by optimumtact** [comment]
@XDTM that's not a bad idea

**Comment by subject217** [comment]
you can color all clothing by splashing the crayon powder reagent onto it but it only works well with white clothes

**Comment by XDTM** [comment]
Might as well throw deafness in too!

Can't wait to see a captain that hires a mime as a full time translator

**Comment by Shdorsh** [comment]
Do we have a deafness quirk to go along with it? That way antags know they can kill me easily.

**Comment by MrDoomBringer** [comment]
@Militaires ree yes you can w/ a grinder
image
tho i realize it looks a lot shittier than these would so ig its fine

**Comment by Militaires** [comment]
@MrDoomBringer oh u mean that
btw I HAVE FINISHED THE QUIRK BLACKLIST THING, now you cant take blind and nearsighted together

**Comment by InsaneKatrina** [comment]
Now we just need a mute quirk to side with blindness to truly embrace the 'nigh-impossible' play mode

**Comment by Shadowflame909** [comment]
Idea: Give them a regular cane to spawn with. But make it a neutral quirk that gives no points. E M E R G E N T

**Comment by Tlaltecuhtli** [comment]
hitting a turf/object/mob with the cane should make it visible for a few seconds

**Comment by Militaires** [comment]
hey pussies i deigned not to remove the tiny spot of vision you get while blind and you're asking for a photographic cane? get the fuck

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -320,6 +320,8 @@
 /datum/quirk/blindness/on_spawn()
 	var/mob/living/carbon/human/H = quirk_holder
 	var/obj/item/clothing/glasses/sunglasses/blindfold/white/glasses = new(get_turf(H))
-	H.put_in_hands(glasses)
-	H.equip_to_slot(glasses, SLOT_GLASSES)
+	if(!H.equip_to_slot(glasses, SLOT_GLASSES)) //if you can't put it on the user's eyes, put it in their hands, otherwise put it on their eyes
+		H.put_in_hands(glasses)
+	else
+		H.equip_to_slot(glasses, SLOT_GLASSES)
```
Comment (Cyberboss) [inline code/datums/traits/negative.dm]: The else isn't necessary

**Comment by Militaires** [comment]
@cyberboss im rarted, done now

**Comment by PKPenguin321** [comment]
dunno why my comment was marked off topic. i have a blind quadriplegic character and would love to have him usable ingame without manually hacking off all of his limbs and jabbing himself in the eyes

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -305,3 +305,22 @@
...
+
+/datum/quirk/blindness/add()
+	quirk_holder.become_blind(ROUNDSTART_TRAIT)
+
+/datum/quirk/blindness/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/obj/item/clothing/glasses/sunglasses/blindfold/white/glasses = new(get_turf(H))
+	if(!H.equip_to_slot(glasses, SLOT_GLASSES)) //if you can't put it on the user's eyes, put it in their hands, otherwise put it on their eyes
+		H.put_in_hands(glasses)
+	H.equip_to_slot(glasses, SLOT_GLASSES)
```
Comment (Cyberboss) [inline code/datums/traits/negative.dm]: BMAN YOU'RE CALLING THE SAME FUNCTION TWICE

Comment (Militaires) [reply code/datums/traits/negative.dm]: Wat u men by dhis.......

**Pull request merged**